tests/Microsoft.Identity.Test.Unit.net45/CoreTests/Telemetry/TelemetryHelperTests.cs

```diff
@@ -39,7 +39,7 @@ public void HandleTelemetryEvents(List<Dictionary<string, string>> events)
             }
         }
 
-        private class _TestEvent : EventBase
```

Comment:
Is this being used somewhere outside of the class? If not, does it need to be internal?